tests/test_job.py

```diff
@@ -21,6 +21,7 @@
 JOB_ID_URL = urljoin(RevAiAPIClient.base_url, 'jobs/{}'.format(JOB_ID))
 JOBS_URL = urljoin(RevAiAPIClient.base_url, 'jobs')
 CUSTOM_VOCAB = [{"phrases": ["word one", "word two"]}]
+LANG = 'en'
```

Comment:
relatively minor feedback - In general I encourage devs to not use abbreviations. It helps make variable names exceptionally clear
```suggestion
LANGUAGE = 'en'
```